src/content/migrate/4.md

```diff
@@ -1,9 +1,14 @@
 ---
 title: To v4 from v3
-sort: 1
+group: Migrate
+sort: 3
 contributors:
-  - sokra
-  - EugeneHlushko
+  - [sokra]
+  - [EugeneHlushko]
+  - [Kolhar730]
+related:
+  - title: To v2 or v3 from v1
+    url: [https://webpack.js.org/migrate/3/]
```

Comment:
Please see other .md files, we don't use square brakcets for the links, they are given as a placeholder in the contributors guide
```suggestion
    url: https://webpack.js.org/migrate/3/
```